_includes/releases/v22.1/v22.1.0-beta.5.md

```diff
@@ -0,0 +1,51 @@
+## v22.1.0-beta.5
+
+Release Date: May 3, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-1-0-beta-5-security-updates">Security updates</h3>
+
+- `Crdb_internal.reset_sql_stats()` and `crdb_internal.reset_index_usage_stats()` builtins now check if user has the admin role. [#80384][#80384]
+- [#80248][#80248] [7d55af0e6][7d55af0e6]  (Raphael 'kena' Poss <knz@cockroachlabs.com>)
```

Comment:
I'm not sure what's happened here? Should there be text? Can you double check if this should be a note or not?